README.md

```diff
@@ -1,52 +1,167 @@
+
+#### index.html
+```html
+<!doctype html>
+<head>
+  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
+    <script src="script.js"></script>
+</head>
+<body>
+  <p id="sentences">
+    Loading our page...
+  </p>
+</body>
+```
+
+#### html/sentence.html
+```html
+<p>Ajax is a brand of cleaning products, introduced by Colgate-Palmolive in 1947 for a powdered household and industrial cleaner.</p>
+<p>It was one of the company's first major brands.</p>
+```
+
+#### javascript.js
+```javascript
+// We should wait for the page to load before running our ajax request
+$(document).ready(function(){
+  // Now we start the Ajax Get request. The first parameter is the URL with the data.
+  // The second parameter is a function that handles the response.
+  $.get("html/sentence.html", function(response) {
+    // Here we are getting the element on the page with the id of sentences and
+    // insert the response
+    $("#sentences").html(response);
+  });
+});
+```
+Using your terminal run the following command from inside the examples folder.
+
+```
+$ python -m SimpleHTTPServer
+```
+
+This starts a simple server that will serve our files over HTTP. Browse to [http://localhost:8000](http://localhost:8000/) and watch as the Ajax request is made and the new data is added to our web page. Pretty cool! We used the power of Ajax to load data from the ```html/sentence.html```. This same idea can be applied to calling the GitHub API or our Rails application.
+
+##Callbacks
+Ajax follows an asynchronous pattern which makes them non-blocking. This means we don't sit around and wait for the request to finish before running the rest of our code. We set callbacks and then fire and forget. When the request is complete, the callbacks are responsible for deciding what to do with the returned data.
+
+If we look at our last example, the Ajax request completed very quick but this won't always be the case. If we request data from a slow server over a slow internet connection, it might take multiple seconds to get a response. Using a callback allows our code to make our request and continue doing other things until the request completes.
+
+To make this concept stick, let's look at a real world example. When you put food into a microwave, you don't stand there and wait for the food to finish cooking. You probably pick up your phone, look at Instagram, read some text messages and of course, work on [Learn](https://learn.co). Basically, you are doing other things while the microwave takes care of cooking your food. When the food is cooked, the microwave beeps and you remove the food and eat it. This final step of removing the food and eating it is exactly how our callbacks work.
```

Comment:
It's important here to show the power of a callback vs. polling. Mention that when you are microwaving something, you don't check the time every 5 seconds...you just wait for the microwave to beep. Once it beeps (CALLBACK!) you then do something with your food. Same way for callbacks